### PR TITLE
add cqlsh

### PR DESCRIPTION
**Pull request by johanesalxd**
changelog:

- add cqlsh (cassandra client)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,30 +2,33 @@ FROM ubuntu:bionic
 
 RUN apt-get update \
   && apt-get install --no-install-recommends --yes --force-yes  \
-    locales \
```
Comment (arunvelsriram) [inline Dockerfile]: could you revert these tab space changes?

**Comment by arunvelsriram** [comment]
After fixing the comment please squash all the commits.

File: Dockerfile (hunk 1 above)
Comment (johanesalxd) [reply Dockerfile]: cool. modified

**Review by johanesalxd — COMMENTED** [review]
done. i'm rebasing my changes

**Pull request merged**